src/components/Core/AppCore/AppCore.js

```diff
@@ -18,7 +18,7 @@ import ConnectivityWrap from '../../Common/Navigation/Connectivity/ConnectivityW
 import ContentSelectorWrap from '../../Common/Misc/ContentSelectorWrap'
 import ContentSelector from '../../Common/Misc/ContentSelector'
 
-export default function AppCore({ agent }) {
+export default function AppCore() {
   const [configuredOrigin, setConfiguredOrigin] = useState('')
   const [data, setData] = useState({})
   const [status, error, startFetchHandler] = useGetServerStatus()
```

Comment:
Should we also change the comment to reflect the new agent name?

Reply:
sorry which comment do you mean?

Reply:
actually I see there's a few comments dotted around, I'll change those